docs/sources/panels-visualizations/configure-legend/index.md

```diff
@@ -1,9 +1,10 @@
 ---
 aliases:
   - ../../panels/working-with-panels/configure-legend/
-  - ../../visualizations/configure-legend/
+  - ../panels-visualizations/visualizations/configure-legend/
+  - ../panels-visualizations/configure-legend/
```

Comment:
I think this alias is not needed, I believe it refers to the current page location.

```suggestion
```

Reply:
I likely added that one because of our whole "preemptive aliases" convention. But I'm happy to leave it out as I think doing that is confusing.

Reply:
Yeah, especially with relative aliases :)